.github/workflows/rebuild_latest_docs.yml

```diff
@@ -14,11 +14,15 @@ on:
         default: "2.0.0"
         required: true
 
+permissions:
+  id-token: write
+
```

Comment:
Simon, we are trying to keep the permissions very granular in our worflows and I don't think this is necessary as you are adding this permission on the `release-docs` job. Have you tried without it?

Reply:
It seems that if you don't have id-token: write on the caller, it won't pass it down – I kept getting a `id-token` should be none error

Reply:
Thank you for the explanation, Simon! In the future we'll refactor this workflows and we can review this.